File: code/video/MyCloudMusicAndroidJava/app/src/main/java/com/ixuea/courses/mymusic/util/DataUtil.java
package com.ixuea.courses.mymusic.util;

import com.ixuea.courses.mymusic.component.song.model.Song;

import java.util.List;

public class DataUtil {
    public static void changePlayListFlag(List<Song> datum, boolean value) {
        for (Song data : datum) {
            data.setPlayList(value);
        }
    }
}

File: code/video/MyCloudMusicAndroidJava/app/src/main/java/com/ixuea/courses/mymusic/util/DataUtil.kt
package com.ixuea.courses.mymusic.util

import com.ixuea.courses.mymusic.component.song.model.Song

object DataUtil {
    @JvmStatic
    fun changePlayListFlag(datum: List<Song>, value: Boolean) {
        for (data in datum) {
            data.isPlayList = value
        }
    }
}
